pkg/pcap/common.go

```diff
@@ -0,0 +1,185 @@
+package pcap
+
+import (
+	"fmt"
+	"math"
+	"os"
+
+	"github.com/aquasecurity/tracee/pkg/logger"
+	"github.com/aquasecurity/tracee/pkg/utils"
+	"github.com/aquasecurity/tracee/types/trace"
+	"github.com/google/gopacket/layers"
+	"github.com/google/gopacket/pcapgo"
+)
+
+var outputDirectory *os.File
+var fake pcapgo.NgInterface
+
+const (
+	pcapDir     string = "pcap/"
+	pcapProcDir string = pcapDir + "processes/"
+	pcapContDir string = pcapDir + "containers/"
+	pcapCommDir string = pcapDir + "commands/"
+)
+
+const (
+	pcapsToCache   = 100 // number of pcaps to keep cached in LRU cache
+	flushAtPackets = 1   // flush pcap file after X number of packets
+)
+
+// TODO: flush per interval as well (so we can grow flushAtPackets higher)
+
+const AF_INET = 2
+const AF_INET6 = 10
+
+//
+// Functions
+//
+
+func initializeGlobalVars(output *os.File) {
```

Comment:
This can just be the `init()` method of the package.
With regards to the output file, i'm rather adamant about moving to a regular config struct, I will expand further near the relevant code.